docs/current_docs/api/chaining.mdx

```diff
@@ -12,110 +14,441 @@ The Dagger CLI can add follow-up processing to a just-in-time container, essenti
 
 Here is an example of chaining a `Container.withExec()` call to a container returned by a Wolfi container builder Dagger Function, to execute a command that displays the contents of the `/etc/` directory:
 
+<Tabs groupId="shell">
+<TabItem value="System shell">
 ```shell
-dagger -m github.com/shykes/daggerverse/wolfi@v0.1.4 call container with-exec --args="ls","/etc/" stdout
+dagger <<EOF
+github.com/dagger/dagger/modules/wolfi@v0.16.2 \
```

Comment:
Need to remove the trailing `\` from all the `dagger <<EOF` examples

Reply:
Leaving as is because it breaks when the trailing `\` is absent

```
$ dagger <<EOF
github.com/dagger/dagger/modules/wolfi@v0.16.2
  | container
  | with-exec ls /etc/
  | stdout
EOF
✔ connect 0.3s
✔ load module 0.9s

Error: -:2:3: | can only immediately follow a statement
```